composer.json

```diff
@@ -19,15 +19,15 @@
         }
     ],
     "require": {
-        "php":                                  ">=5.3.9",
+        "php":                                  ">=5.5.9",
         "a2lix/translation-form-bundle":        "~2.0",
         "doctrine/collections":                 "~1.2",
-        "doctrine/common":                      "~2.3",
+        "doctrine/common":                      "~2.5",
         "doctrine/doctrine-bundle":             "~1.3",
         "doctrine/doctrine-cache-bundle":       "~1.0",
         "doctrine/doctrine-fixtures-bundle":    "~2.2",
         "doctrine/doctrine-migrations-bundle":  "~1.0",
-        "doctrine/orm":                         "~2.4",
+        "doctrine/orm":                         ">=2.4.8,<2.5",
```

Comment:
Version `2.5.0` has security issue and version `2.5.1` fixes that issue but introduces some bugs that affects our `@products` suite.